style.css

```diff
@@ -0,0 +1,145 @@
+            a:hover {
```

Comment:
There is a lot of style declarations in this file.
Could more of Bootstrap be used so this file is smaller, and Bootstrap is utilised more?